jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -344,6 +330,21 @@ def calculate_moment(self, add_data=True):
 
         return self.moment
 
+    @property
+    def moment_zero_unit(self):
+        if self.spectrum_viewer.state.x_display_unit is not None:
+            return (
+                u.Unit(self.app._get_display_unit('sb')) *
+                self.spectrum_viewer.state.x_display_unit
+            )
+        return ''
+
+    @property
+    def spectral_unit_selected(self):
+        # NOTE: should we add this logic to app._get_display_unit?
+        unit_conv = self.app.get_tray_item_from_name('g-unit-conversion')
+        return unit_conv.spectral_unit.selected
```

Comment:
I think this is available from `_get_display_unit` now if you want to make the change here/now